doc/tutorial/using_gpu.txt

```diff
@@ -453,11 +109,11 @@ Returning a Handle to Device-Allocated Data
 By default functions that execute on the GPU still return a standard
 numpy ndarray.  A transfer operation is inserted just before the
 results are returned to ensure a consistent interface with CPU code.
-This allows changing the deivce some code runs on by only replacing
+This allows changing the device some code runs on by only replacing
 the value of the ``device`` flag without touching the code.
 
 If you don't mind a loss of flexibility, you can ask theano to return
-the GPU object directly.  The following code is modifed to do just that.
+the GPU object directly.  The following code is modified to do just that.
 
 .. testcode::
 
```

Comment:
I changed the code and it runs fine.

However, there is no more speedup from explicitely transfering the variable to GPU. On a GTX 680, both examples from the tutorial take about 2.2 sec.